markdown/about.md

```diff
@@ -4,21 +4,75 @@ The sanger-tol project is the informatics branch of the Tree of Life programme a
 Here you can read how we organise ourselves, how we are funded and how the sanger-tol project was started.
 
 We welcome external contributions and collaboration on our projects.
-If you'd like to be involved, [drop us a message](https://github.com/sanger-tol/pipelines-website/issues/new?assignees=priyanka-surana%2Cmuffato&labels=connect&projects=&template=contact_us.yaml&title=%5BContact+Us%5D%3A+) or comment directly on an existing GitHub issue.
+If you'd like to be involved, [drop us a message](https://github.com/sanger-tol/pipelines-website/issues/new?assignees=muffato%2Cmuffato&labels=connect&projects=&template=contact_us.yaml&title=%5BContact+Us%5D%3A+) or comment directly on an existing GitHub issue.
 
 Please note that all community members are expected to adhere to our [code of conduct](/code_of_conduct), which is adopted from [nf-core](https://nf-co.re).
 
 ## Informatics Infrastructure team {#it}
 
-The [Informatics Infrastructure team](/it) provides support for the production of reference genome assemblies and large-scale genome analyses in the Tree of Life programme, and helps with the management and use of informatics resources and digital solutions. Current members are listed below:
+### Introduction {#it-intro}
```

Comment:
This subtitile can be removed. There are two **Introduction** in ths page currently and it seems too much.

Reply:
Done